File: GTTS/INL.VettingService.Models/GetPersonsVettings_Result.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace INL.VettingService.Models
{
    public class GetPersonsVettings_Result : IGetPersonsVettings_Result
    {
        public List<GetPersonsVetting_Item> VettingCollection { get; set; }
    }
}

File: GTTS/GTTS.Web/ClientApp/src/app/models/INL.VettingService.Models/get-persons-vettings_result.ts


import { GetPersonsVetting_Item } from './get-persons-vetting_item';
import { IGetPersonsVettings_Result } from './iget-persons-vettings_result';

export class GetPersonsVettings_Result implements IGetPersonsVettings_Result {
  
	public VettingCollection?: GetPersonsVetting_Item[];
  
}


